PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install CrowdStrike Falcon Agent.md

```diff
@@ -49,14 +166,15 @@ exit $installerProcess.ExitCode
 
 #### Description
 
-This command will download and install the CrowdStrike Falcon Agent to the device if it isn't already installed.
+This command will download and install the CrowdStrike Falcon Agent to the device if it isn't already installed. The command will leverage CrowdStrike's API to find and download the latest version of the Falcon Agent onto the local machine.
+
+Follow the instructions from the [Installing the CrowdStrike Falcon Agent KB](https://support.jumpcloud.com/s/article/Installing-the-Crowdstrike-Falcon-Agent#InstallWindows)
 
 In order to use this command:
 
-1. Download the CrowdStrike Falcon Agent installer and host it at a URL that your devices can access.
-2. Edit the first two lines of the script to include your Customer ID (with checksum value) and the URL where you are hosting the installer.
+1. Create a CrowdStrike API Client with the "SENSOR DOWNLOAD" Read scope and make note of the ClientID and ClientSecret Refer to CrowdStrike's article [Getting Access to the CrowdStrike API](https://www.crowdstrike.com/blog/tech-center/get-access-falcon-apis/) for further information
+2. Set the 3 variables (CSBaseAddress, CSClientID, CSClientSecret) to their respective values for your CrowdStrike API Client
 3. Extend the command timeout to a value that makes sense in your environment. The suggested command timeout for an environment with average network speeds on devices with average computing power is 10 minutes. Note that the command may timeout with a 124 error code in the command result window if not extended, but the script will continue to run.
```

Comment:
Yes the first time I ran this I did see the 124 error but the command did complete and the host showed up in my console

Reply:
I got the same exact issue